npmprune.sh

```diff
@@ -47,12 +47,15 @@ PATTERNS="
 	*tslint*
 	*vscode*
 	*wallaby*
-	authors
-	changelog
+	authors*
```

Comment:
Too broad because it can match a file named `authors.js`.  
Same for changelog, contributors and license.